### PR TITLE
specs task controller

### PR DESCRIPTION
**Pull request by ShevchukTania**
**STATUS**
READY

**DESCRIPTION**

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [ ] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [ ] PR is reviewed manually again (to make sure you have 100% ready code)
- [ ] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs

### CODE REVIEW

File: spec/controllers/account/tasks_controller_spec.rb [hunk 1]
```diff
@@ -57,6 +57,18 @@
...
+    it 'should create section' do
+      post :create, params: {
+        project_id: project.id,
+        task: {
+          title: Faker::Lorem.sentence,
+          description: Faker::Lorem.paragraph,
+          section: true
+        }
+      }
+       expect(Task.last.section).to be true
```
Comment (romaslmd) [inline spec/controllers/account/tasks_controller_spec.rb]: nope

Comment (romaslmd) [reply spec/controllers/account/tasks_controller_spec.rb]: `expect to change(<smth>).by(1)`

Comment (ShevchukTania) [reply spec/controllers/account/tasks_controller_spec.rb]: подумала, що можна не тестити ще раз те, що вже відтестив інший тест. просто перевірити, що те, що створилось, є секцією

File: spec/controllers/account/tasks_controller_spec.rb [hunk 2]
```diff
@@ -58,6 +58,20 @@
...
+      post :create, params: {
+        project_id: project.id,
+        task: {
+          title: Faker::Lorem.sentence,
+          description: Faker::Lorem.paragraph,
+          section: true
+        }
+      }
+      expect{ post :create, params: { project_id: project.id, task: { title: Faker::Lorem.sentence,
+        description: Faker::Lorem.paragraph, section: true }}}.to change(Task, :count).by(1)
```
Comment (romaslmd) [inline spec/controllers/account/tasks_controller_spec.rb]: still not good..
why wouldn't u try `change { Task.all.section.count }.by(1)` or `change { Task.all.where(section: true).count }.by(1)`

File: spec/controllers/account/tasks_controller_spec.rb [hunk 3]
```diff
@@ -58,6 +56,19 @@
...
+      post :create, params: {
+        project_id: project.id,
+        task: {
+          title: Faker::Lorem.sentence,
+          description: Faker::Lorem.paragraph,
+          section: true
+        }
+      }
+      expect{ post :create, params: { project_id: project.id, task: { title: Faker::Lorem.sentence,
+        description: Faker::Lorem.paragraph, section: true }}}.to change(Task.all.where(section: true), :count).by(1)
```
Comment (romaslmd) [inline spec/controllers/account/tasks_controller_spec.rb:69]: not very good, but will do
better to use blocks with scopes

**Review by romaslmd — APPROVED** [review]
looks ok

**Pull request merged**